modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceWishList.testcase

```diff
@@ -591,6 +591,131 @@ definition {
+				wishListName = "Default");
+		}
+
+		task ("When the buyer import orders from wish list") {
+			CommerceAcceleratorsInitializer.createNewOrderFromPendingOrders();
+
+			CommerceOrders.importOrderFromType(orderType = "Wish Lists");
+
+			SelectFrameTop();
+
+			IFrame.selectModalFrame();
+
+			ClickNoError(
+				key_itemName = "Default",
+				locator1 = "CommerceEntry#SELECT_ITEM_FROM_LIST");
+
+			SelectFrameTop();
+
+			IFrame.selectModalFrame();
+		}
+
+		task ("And can see data set is filled with the following data in the preview modal and only eigth items are displayed") {
+			Pagination.changePagination(itemsPerPage = "8 items");
+
+			CommerceEntry.viewPaginationResults(results = "Showing 1 to 8 of 13");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "Premium Brake Pads",
+				productQuantity = 1,
+				productSku = "MIN93019",
+				totalPrice = "29.00",
+				unitPrice = "29.00");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "Brake Rotors",
+				productQuantity = 1,
+				productSku = "MIN93020",
+				totalPrice = "40.00",
+				unitPrice = "40.00");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "Calipers",
+				productQuantity = 1,
+				productSku = "MIN93021",
+				totalPrice = "90.00",
+				unitPrice = "90.00");
+		}
+
+		task ("When the user go to next page") {
+			Click(locator1 = "ACUtils#NEXT_PAGES");
+		}
+
+		task ("Then the remaining items are displayed") {
+			CommerceEntry.viewPaginationResults(results = "Showing 9 to 13 of 13");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "ABS Sensor",
+				productQuantity = 1,
+				productSku = "MIN93015",
+				totalPrice = "50.00",
+				unitPrice = "50.00");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "U-Joint",
+				productQuantity = 1,
+				productSku = "MIN55861",
+				totalPrice = "24.00",
+				unitPrice = "24.00");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "Torque Converters",
+				productQuantity = 1,
+				productSku = "MIN55859",
+				totalPrice = "34.00",
+				unitPrice = "34.00");
+
+			AssertElementNotPresent(
+				key_columnNumber = 1,
+				key_productName = "Calipers",
+				key_value = "MIN93021",
+				locator1 = "CommerceEntry#TABLE_SEARCH_BY_NAME_AND_COLUMN_CONTENT");
+
+			AssertElementNotPresent(
+				key_columnNumber = 1,
+				key_productName = "Brake Rotors",
+				key_value = "MIN93020",
+				locator1 = "CommerceEntry#TABLE_SEARCH_BY_NAME_AND_COLUMN_CONTENT");
+
+			AssertElementNotPresent(
+				key_columnNumber = 1,
+				key_productName = "Premium Brake Pads",
+				key_value = "MIN93019",
+				locator1 = "CommerceEntry#TABLE_SEARCH_BY_NAME_AND_COLUMN_CONTENT");
+
+			IFrame.closeFrame();
+		}
```

Comment:
The assertion of the previous page arrow is missing. Also paginating by directly clicking on a page number is missing